src/sentry/conf/server.py

```diff
@@ -1558,6 +1558,8 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     "organizations:slack-escape-messages": False,
     # If true, allow to create/use org auth tokens
     "organizations:org-auth-tokens": False,
+    # Enables commenting on PRs from the Sentry comment bot.
```

Comment:
alphabetical order? I know its a little messy tho and there's like 3 diff parts that are each in their own alphabetical order